plugins/maintenance/src/test/java/org/apache/cloudstack/maintenance/ManagementServerMaintenanceManagerImplTest.java

```diff
@@ -154,7 +154,7 @@ public void prepareForShutdownCmdOtherMsHostsInPreparingState() {
         ManagementServerHostVO msHost2 = mock(ManagementServerHostVO.class);
         List<ManagementServerHostVO> msHostList = new ArrayList<>();
         msHostList.add(msHost2);
-        Mockito.when(msHostDao.listBy(any())).thenReturn(msHostList);
+        Mockito.lenient().when(msHostDao.listBy(any())).thenReturn(msHostList);
```

Comment:
iirc mockito was throwing errors earlier when an unnecessary mocks was setup. Why is `lenient()` required now?